footprint/cortex-m/run_suite.py

```
@@ -0,0 +1,234 @@
+                output = run_qemu(target, example, features)
+            except subprocess.TimeoutExpired:
+                print("    TIMEOUT", file=sys.stderr)
+                failures.append(f"Timeout: {example} on {label}")
+                continue
+
+            accepted = "rsa ACCEPT" in output
+            metric = parse_metric(output)
+            text_size = get_text_size(target, example, features)
+
+            if not metric:
+                print(f"    METRIC line missing", file=sys.stderr)
```

Ruff (0.15.13)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)